src/client/components/RenderIFrame.tsx

```diff
@@ -0,0 +1,46 @@
+import '../styles/IframeView.css'
+
+import React, { useEffect, useState } from 'react'
+
+interface IRenderIFrameProps {
+    src: string
+}
+
+const RenderIFrame = (props: IRenderIFrameProps) => {
+    const [scale, setScale] = useState(1)
+
+    useEffect(() => {
+        const handleResize = () => {
+            let scaleX = window.innerWidth / 1920
+            let scaleY = window.innerHeight / 1200
+
+            // Scale to the smallest part of view
+            setScale(scaleX)
+            if (scaleY < scaleX) {
+                setScale(scaleY)
+            }

```

Comment:
Wouldn't it be better to use an else-branch? This way we would only trigger a re-render once instead of twice in the case where scaleY < scaleX.
```suggestion
            } else {
                setScale(scaleX)
            }
```